h2o-core/src/main/java/water/rapids/ShuffleVec.java

```diff
@@ -0,0 +1,76 @@
+package water.rapids;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Vec;
+import water.fvec.Frame;
+import water.util.FrameUtils;
+import water.util.RandomUtils;
+
+import java.util.Random;
+
+import static water.util.RandomUtils.getRNG;
+
+public class ShuffleVec  {
+    
+    Frame _fr;
+    String _col_name;
+    
+    
+    
+    public ShuffleVec(Frame fr) { 
+        _fr = fr;    
+    }
+    
+    /* create a frame with shuffled columns */
+    public /*Frame*/ void ShuffleFrameVecs(int features_no, Frame out){
```

Comment:
How `Frame out` is planned to use here?

Reply:
Initial Idea was a bit different. It is removed